### PR TITLE
Merge branch 'main' of https://github.com/Oluwasetemi/altschool-opensource-names

### PR DESCRIPTION
**Pull request by kkins25**

### CODE REVIEW

File: names.txt [hunk 1]
```diff
@@ -42,3 +42,8 @@ Adetunji Igbatayo
 Taiye Kadiri
 Oluwakunmi Obaditan
 Clinton Mekwunye
+<<<<<<< HEAD
```
Comment (oseunabiola) [inline names.txt:45]: Kindly remove all lines that are not for names.

**Review by oseunabiola — CHANGES_REQUESTED** [review]
Kindly remove all lines that are not for names.

**Comment by tobisupreme** [comment]
Hi Kingsley! 😁

We are here for you for the next 6 hours before we close this PR so you can initiate a fresh one which will be easier to merge.

Please refer to the updated README 🙏

**Comment by Oluwasetemi** [comment]
Please I am sorry for all the trouble we have caused you but count it all joy. Your PR is about to be merged and you will be able to see your name on this [url](https://altschool-opensource-names.oluwasetemi.dev/). 

To enable me to do this, **READ the README again carefully**, update your code with the latest and push the update. I am here for you for the next **6 hours** before I close the PR so you can initiate a fresh PR which will be easier to merge.

**Pull request closed without merging**